.github/workflows/dependency-check.yml

```diff
@@ -1,6 +1,10 @@
 name: Dependency Vulnerability Check
 
-on: [push]
+on: 
+  push:
+    paths:
```

Comment:
We could isolate this even more to only run this check of the go.mod/sum changes.

Reply:
Only one go.mod/sum exists, dont need to use globs here

Reply:
Hmm, I guess the thought was in case we had other subdirs later on with their own go.mod files? Thoughts?